android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -396,6 +397,16 @@ public void call_getStats(String uuid,  Promise promise) {
     }
   }
 
+  @ReactMethod
+  public void call_sendMessage(String uuid, CallMessage message, Promise promise) {
+    final CallRecord callRecord = validateCallRecord(reactContext, UUID.fromString(uuid), promise);
+
+    if (null != callRecord) {
```

Comment:
What is the situation where there isn't a call record? 
* When the message comes via call-invite there should exist a call record
* When the message comes via a call, there should be a call record.

it seems like it should instead have a check to throw an assert if the callRecord is missing
Objects.requireNonNull(...)